shaders/src/common_getters.glsl

```diff
@@ -46,6 +46,11 @@ highp mat4 getViewFromClipMatrix() {
     return frameUniforms.viewFromClipMatrix;
 }
 
+/** @public-api */
+highp mat4 getEyeFromViewMatrix() {
+    return frameUniforms.eyeFromViewMatrix[getEyeIndex()];
+}
```

Comment:
I think for this one we want a method that take the eyeIndex as a parameter. Maybe add an override. I think as part of this PR we can also add a version of getClipFromWorldMatrix(int) that takes the eyeIndex.